src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss

```diff
@@ -12,7 +12,269 @@
     padding-top: 200px;
 }
 
+.img404 {
+    margin-bottom: 35px;
+
+    position: relative;
+    z-index: 1;
+}
+
+.text404 {
+    position: relative;
+    z-index: 99;
+}
+
 .honeyLost {
     font-family: "geometriaBold";
     font-size: 4rem;
+    line-height: 1em;
+
+    position: relative;
+    z-index: 1;
+}
+
+.tears {
+    position: absolute;
+    left: 50%;
+    top: 50px;
+    z-index: 2;
+}
+
+.tearDrop {
+    @mixin animation_name(tears);
+    @mixin animation_duration(4s);
+    @mixin animation_timing_function(linear);
+}
+
+@-webkit-keyframes tears {
+    0% {
+        -webkit-transform: scale(.25) translate(0, 0);
+    }
+    100% {
+        -webkit-transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+@keyframes tears {
+    0% {
+        -moz-transform: scale(.25) translate(0, 0);
+        -ms-transform: scale(.25) translate(0, 0);
+        -o-transform: scale(.25) translate(0, 0);
+        transform: scale(.25) translate(0, 0);
+    }
+
+    100% {
+        -moz-transform: scale(.75) translate(0, 1000px);
+        -ms-transform: scale(.75) translate(0, 1000px);
+        -o-transform: scale(.75) translate(0, 1000px);
+        transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+.tears2 {
+    margin-left: -5px;
+
+    position: absolute;
+    left: 50%;
+    top: 50px;
+    z-index: 2;
+}
+
+.tearDrop2 {
+    @mixin animation_name(tears2);
+    @mixin animation_duration(4s);
+    @mixin animation_delay(.5s);
+    @mixin animation_timing_function(linear);
+}
+
+@-webkit-keyframes tears2 {
+    0% {
+        -webkit-transform: scale(.25) translate(0, 0);
+    }
+    100% {
+        -webkit-transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+@keyframes tears2 {
+    0% {
+        -moz-transform: scale(.25) translate(0, 0);
+        -ms-transform: scale(.25) translate(0, 0);
+        -o-transform: scale(.25) translate(0, 0);
+        transform: scale(.25) translate(0, 0);
+    }
+    100% {
+        -moz-transform: scale(.75) translate(0, 1000px);
+        -ms-transform: scale(.75) translate(0, 1000px);
+        -o-transform: scale(.75) translate(0, 1000px);
+        transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+.tears3 {
+    margin-left: -25px;
+
+    position: absolute;
+    left: 50%;
+    top: 50px;
+    z-index: 2;
+}
+
+.tearDrop3 {
+    @mixin animation_name(tears3);
+    @mixin animation_duration(4.25s);
+    @mixin animation_delay(.25s);
+    @mixin animation_timing_function(linear);
+}
+
+@-webkit-keyframes tears3 {
+    0% {
+        -webkit-transform: scale(.25) translate(0, 0);
+    }
+    100% {
+        -webkit-transform: scale(.75) translate(0, 1000px);
+    }
+}
+
+@keyframes tears3 {
+    0% {
+        -moz-transform: scale(.25) translate(0, 0);
```

Comment:
This repetition has nothing to do with mixins. Mixins are about taking one thing and adding vendor prefixes. This is more of a variable concern, and there is no need for a variable here I think.